msteams-platform/apps-in-teams-meetings/build-apps-for-anonymous-user.md

```diff
@@ -9,44 +9,64 @@ ms.localizationpriority: medium
+
+Anonymous users can't be authenticated through Azure Active Directory (Azure AD) authentication or `getAuthToken` from the client SDK as they aren't Azure AD accounts. If you need to authenticate anonymous users, your app must identify anonymous users and provide an alternative authentication experience in the meetings. You can determine if a user is anonymous by validating [user's context](#in-meeting-getcontext-from-teams-client-library).
 
 ## Admin setting for anonymous user app interaction
 
-Teams admins can use the admin portal to enable or disable anonymous user app interaction for the entire tenant. This setting is enabled by default. For more information, see [allow anonymous users to interact with apps in meetings](/microsoftteams/meeting-settings-in-teams).
+Teams admins can use the Teams admin center to enable or disable anonymous user app interaction for the entire tenant. If your app needs to be accessed by anonymous users, ensure that the tenant admins enable the anonymous user app interaction. This setting is enabled by default. For more information, see [allow anonymous users to interact with apps in meetings](/microsoftteams/meeting-settings-in-teams).
+
+To test your apps experience for anonymous users, select the URL in the meeting invite and join the meeting from a private browser window.
 
 ## In-Meeting getContext from Teams client library
 
-Apps receive the following info for an anonymous user when they call the `getContext` API from the shared app stage. You can recognize anonymous users by checking for a `userLicenseType` value of **Unknown**.
+Apps receive the following info for an anonymous user when they call the `getContext` API from the shared app stage. You can recognize anonymous users by checking for a `userLicenseType` value of `Anonymous`.
 
 ```csharp
-"userObjectId": "8:anon:<GUID1>",
-"userLicenseType": "Unknown",
-"loginHint": "8:teamsvisitor:<ID>",
-"userPrincipalName": "8:teamsvisitor:<ID>",
-"tid": "<meeting organizer tenant ID>"
+"userObjectId": "",
+"userLicenseType": "Anonymous",
+"loginHint": "",
+"userPrincipalName": "",
 ```
 
 | **Property name** | **Description** |
 | --- | --- |
-| `userObjectId` | Unique generated value for the anonymous user. This value cannot be used in calls to Graph APIs. |
-| `userLicenseType` | `Unknown`, represents anonymous user. |
-| `loginHint` | Unique generated value. This value cannot be used as a hint in login flows. |
-| `userPrincipalName` | Unique generated value. This value cannot be used in calls to Graph APIs. |
-| `tid` | Tenant ID of the meeting organizer. |
+| `userLicenseType` | `Anonymous` represents anonymous user. |
 
 > [!NOTE]
-> When an anonymous user joins a meeting, a new user ID is generated. Whenever the anonymous user re-joins a meeting, a different user ID is generated.
+> When an anonymous user joins a meeting, a new user ID is generated. Whenever the anonymous user rejoins a meeting, a different user ID is generated.
```

Comment:
Does this belong in this section? There's no user ID in tab context anymore.

Reply:
Removed the Note